psydac/api/fem.py

```diff
@@ -916,16 +1029,60 @@ def __init__(self, expr, kernel_expr, *args, **kwargs):
         if vector_space.parallel and vector_space.cart.num_threads>1:
             num_threads = vector_space.cart._num_threads
 
+        if vector_space.parallel and vector_space.cart.is_comm_null:
+            self._free_args = ()
+            self._func      = do_nothing
+            self._args      = ()
+            self._expr      = expr
+            self._comm      = domain_h.comm
+            return
```

Comment:
Please explain the purpose of this `if` block

Reply:
done in [ec10ae7](https://github.com/pyccel/psydac/pull/213/commits/ec10ae70e2106c6cc339a5aa2ae5ac81c351a7f6)